src/Microsoft.AspNet.Mvc.Core/Formatters/HttpNotAcceptableOutputFormatter.cs

```diff
@@ -1,40 +0,0 @@
-// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Threading.Tasks;
-using Microsoft.AspNet.WebUtilities;
-using Microsoft.Net.Http.Headers;
-
-namespace Microsoft.AspNet.Mvc
-{
-    /// <summary>
-    /// A formatter which does not have a supported content type and selects itself if no content type is passed to it.
-    /// Sets the status code to 406 if selected.
-    /// </summary>
-    public class HttpNotAcceptableOutputFormatter : IOutputFormatter
```

Comment:
How does a user control the output for a 406? If for example he wants to add an extra header with allowed formats.